GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -343,8 +399,27 @@ end );
 ##
 InstallGlobalFunction( ADD_GRADED_IS_WELL_DEFINED_FOR_MORPHISM_RIGHT,
 
-  function( category )
-    ADD_GRADED_IS_WELL_DEFINED_FOR_MORPHISM_LEFT( category );
+function( category )
+    AddIsWellDefinedForMorphisms( category,
+      function( morphism )
+        local matrix, matrix_entries, matrix_entries_degrees, required_degrees;
+        
+        if not IsWellDefined( UnderlyingPresentationMorphism( morphism ) ) then
+            return false;
+        fi;
+        
+        matrix := UnderlyingMatrix( UnderlyingPresentationMorphism( morphism ) );
+        
+        matrix_entries := Flat( EntriesOfHomalgMatrixAsListList( matrix ) );
+        
+        matrix_entries_degrees := Flat( DegreesOfEntries( matrix ) );
+        
+        required_degrees := Flat( List( GeneratorDegrees( Range( morphism ) ), i -> 
+                                    List( GeneratorDegrees( Source( morphism ) ), j -> j - i ) ) );
```

Comment:
@sebasguts 
Remark by Kamal:
the difference between `ADD_GRADED_IS_WELL_DEFINED_FOR_MORPHISM_LEFT`
and his function lies in `j-i` and `i-j`

Reply:
Thx :)